companion.py

```diff
@@ -189,6 +190,22 @@ def __init__(self, *args) -> None:
             self.args = (_('Error: Frontier server is lagging'),)
 
 
+class NoMonitorStation(Exception):
```

Comment:
Does this maybe want to inherit from ServerLagging

Reply:
That feels misleading.  In this case Frontier CAPI server isn't found to be lagging.  This is literally "we forgot the monitor.station, so can't even check".

Reply:
thats fair